### PR TITLE
Update connect-to-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Section was missing the import-module cmdlet

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8116

File: microsoft-365/enterprise/connect-to-microsoft-365-powershell.md [hunk 1]
```diff
@@ -96,6 +96,12 @@ Are you sure you want to install the modules from 'PSGallery'?
 
 Answer **Yes** or **Yes to All** to continue with the installation.
 
+3.  Run this command to import the module
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/connect-to-microsoft-365-powershell.md]:
```suggestion
3.  Run this command to import the module:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**